cif_printout_Python: tests/inputs/test-biblio-stray-vals.cif(1): ERROR, stray CIF values at the beginning of the input file.
